test_restart_ioc_when_pv_in_alarm.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from genie_python.genie_startup import *
+from general.utilities.restart_ioc_when_pv_in_alarm import restart_ioc_when_pv_in_alarm
+from utilities.utilities import load_config_if_not_already_loaded
+
+
+class TestRestartIocWhenPvInAlarm(unittest.TestCase):
+    """
+    Tests for the `restart_ioc_when_pv_in_alarm` script.
+    """
+    def setUp(self) -> None:
+        g.set_instrument(None, import_instrument_init=False)
+        load_config_if_not_already_loaded("test_restart_ioc_when_pv_in_alarm")
+        
+
+    def test_WHEN_ioc_in_alarm_THEN_ioc_restarted(self):
+        self.assertEqual(g.get_pv("SIMPLE:MBBI", is_local=True), "HAPPY")
```

Comment:
Depending on which system test happens to run before this one, the block _could_ have a different value here. Can we do an explicit set to `HAPPY` first just to ensure this doesn't happen?